python/py-userpath/Portfile

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-userpath
+version             1.3.0
+categories-append   devel
+license             {MIT Apache}
+maintainers         @0az openmaintainer
+supported_archs     noarch
+platforms           darwin
+
+description         A tool and library for adding locations to user PATH
+long_description    ${description}
+
+homepage            https://github.com/mitsuhiko/click
+master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
+distname            userpath-${version}
```

Comment:
the fields ```distname``` and ```master_sites``` are set automatically to PyPI if you use the python PG, so please remove these redundant lines.